src/Microsoft.OData.Client/BulkUpdateSaveResult.cs

```diff
@@ -28,6 +31,29 @@ internal class BulkUpdateSaveResult : BaseSaveResult
         /// </summary>
         private readonly BulkUpdateGraph bulkUpdateGraph;
 
+        /// <summary> Link descriptors for some descriptors.</summary>

```

Comment:
The use of "some descriptors" doesn't help the person understand which descriptors end up here... Is it those associated with updated objects?